src/sdgio-edit.owl

```diff
@@ -3293,6 +3307,45 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020299> "increased capacity for innovation in partner development organizations and in poor and vulnerable communities"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020299>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020302> (adaptation to and mitigation of climate change through agricultural research)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020302> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they improve adaptation to and mitigation of climate change through agricultural research.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020302> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020302> "2019-05-08T21:18:39Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020302> "All research and development activities need to build in resilience to climate shocks and a focus on adaptation to and mitigation of climate change. Agriculture including forestry could be the most efficient contributor to the reduction of greenhouse gases through storage in the biomass and the healthy soils. CGIAR is committed to devoting
+at least 60% of its research to those issues.")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020302> "adaptation to and mitigation of climate change through agricultural research"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020302>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020303> (achievement of gender equity and youth employment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020303> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they improve gender equity, especially concerning poor women, and youth inclusion employment.")
```

Comment:
You don't really need definitions for instances, and the ones you have sound more like category definitions. You can turn these into descriptions rather than definitions by extending the comments.

Reply:
That also works for me. However, for the instance in the SLO class we gave definitions. Again, I just wonder whether we should find a uniform way to deal with the high-level DOs (SLOs and CCOs).

Reply:
I extended the comment and removed the definition. I am not sure that I did exactly what you meant though.

Reply:
I suppose that I need to do the same for all the CCOs. So, I will do that now.

Reply:
I moved the xref AP of the definitions to the comments.